rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -231,13 +231,15 @@ def _delete_acl(self, lswitches):
     def _create_networks(self, network_create_args):
         physnet = network_create_args.get("physical_network", None)
         lswitches = self._create_lswitches(network_create_args)
+        batch = network_create_args.get("batch", 1)
```

Comment:
suggest to use the number of lswitches as default value:
batch = network_create_args.get("batch", len(lswitches))